test/lit.cfg

```diff
@@ -1339,36 +1338,22 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
     config.target_runtime = "native"
     config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
     config.target_sdk_name = "android"
-    android_link_paths_opt = "-L {} -L {} -L {}".format(
-        shell_quote(make_path(
-            config.android_ndk_path, "sources", "cxx-stl", "llvm-libc++",
-            "libs", ndk_platform_tuple)),
+    android_link_paths_opt = "-L {} -L {}".format(
         shell_quote(make_path(
             toolchain_directory, "lib", "gcc", ndk_platform_triple,
             "{}.x".format(config.android_ndk_gcc_version))),
         shell_quote(make_path(
             toolchain_directory, ndk_platform_triple, "lib")))
```

Comment:
This directory doesn't seem to contain anything useful. If I am doing the substitutions right, it expands to `<ndk>/toolchains/llvm/prebuilt/<host>/aarch64-android-linux/lib`. In my copy it only has a `ldscripts` directory.

Reply:
That's all there was in the directory used before this pull too, at least with NDK 21, ie it used to search in `<ndk>/toolchains/aarch64-linux-android-4.9/prebuilt/linux-x86_64/aarch64-linux-android/lib` instead. I wasn't sure if that was needed for anything, so I left it in: would you like me to remove it?

Reply:
Removed it in latest commit.